File: src/main/java/csorba/norberth/lab1/models/Espresso.java
package csorba.norberth.lab1.models;

import java.util.List;

public class Espresso extends Coffee {
    public Espresso(List<Ingredient> ingredients) {
        super(ingredients);
    }

    @Override
    public String toString() {
        return super.toString() + " > Espresso";
    }
}

File: Laboratoare_gbir2109/DP_Lab1_Homework/DP_Lab1_Homework/Models/Espresso.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DP_Lab1_Homework.Models
{
    public class Espresso : BaseCoffee
    {
        public Espresso()
        {
            Ingredients = new List<Ingredient>();
            CoffeeType = Coffee.Espresso;
        }
    }
}
